scripts/helm-init

```diff
@@ -64,5 +64,19 @@ helm-init() {
     helm repo update
 }
 
+setup-gitlab-agent() {
+    if [ -n "${GITLAB_AGENT_PROJECT:-}" ] && [ -n "${GITLAB_AGENT_ID:-}" ] && [ "${DISABLE_GITLAB_AGENT:-0}" != "1" ]; then
+        echo "The deployment will use the GitLab Agent."
+        echo "Switching Kubernetes context to use the context provided by the GitLab Agent."
```

Comment:
print here the name of the used context

Reply:
The `kubectl config use-context` prints the switch context message. Here an example:

```bash
Switched to context "<GITLAB_AGENT_PROJECT>:<GITLAB_AGENT_ID>".
```